src/JuMP.jl

```diff
@@ -633,6 +642,21 @@ function MOI.get(model::Model, attr::MOI.AbstractModelAttribute)
         MOI.get(backend(model), attr)
     end
 end
+"""
+    get(model::Model, attr::MathOptInterface.AbstractOptimizerAttribute)
+
+Return the value of the attribute `attr` from the model's MOI backend.
+"""
+function MOI.get(model::Model, attr::MOI.AbstractOptimizerAttribute)
+    if MOI.is_set_by_optimize(attr) &&
+       !(attr isa MOI.TerminationStatus) && # Before `optimize!` is called, the
```

Comment:
You can remove this because `MOI.TerminationStatus` is not an optimizer attribute

Reply:
True. 30f020c7a71eb2357af7117fa2e46b5afff4981a should be fine now.